cmd/virt-handler/virt-handler.go

```diff
@@ -150,6 +153,28 @@ func main() {
 	ws := new(restful.WebService)
 	ws.Route(ws.GET("/api/v1/namespaces/{namespace}/vms/{name}/console").To(console.Console))
 	restful.DefaultContainer.Add(ws)
+
+	labelSelector, err := labels.Parse("daemon in (virt-handler)")
+	if err != nil {
+		fmt.Printf("Can't parse label selector: %s\n", err.Error())
+		return
+	}
+	pods, err := coreClient.CoreV1().Pods(k8sv1.NamespaceDefault).List(meta_v1.ListOptions{LabelSelector: labelSelector.String()})
+	if err != nil {
+		fmt.Printf("Can't find virt-handler pod: %s\n", err.Error())
+		return
+	}
+
+	virtHandlerUrl := pods.Items[0].GetSelfLink()
+	fmt.Println(virtHandlerUrl)
+	_, err = restClient.
+		Patch(types.JSONPatchType).
+		AbsPath(virtHandlerUrl).
+		Body([]byte(fmt.Sprintf("[{\"op\": \"add\", \"path\": \"/metadata/annotations/virtHandlerPort\", \"value\": \"%d\"}]", *port))).Do().Raw()
```

Comment:
could you use the `kubevirt.io/` prefix for the annotation?

Reply:
No, I tried to do with `kubevirt.io/` prefix, but it doesn't work.
Yes, `replace` could be instead of the `add`.